etc/config/fortran.amazon.properties

```diff
@@ -690,29 +690,24 @@ compiler.fs390xg1420.demangler=/opt/compiler-explorer/s390x/gcc-14.2.0/s390x-ibm
 
 ###############################
 # LLVM Flang for X86
-group.clang_llvmflang.compilers=flangtrunk:flangtrunknew:flangtrunknew-fc
+group.clang_llvmflang.compilers=flangtrunk:flangtrunk-fc
 group.clang_llvmflang.groupName=LLVM-FLANG x86-64
 group.clang_llvmflang.compilerType=flang
 
-compiler.flangtrunk.exe=/opt/compiler-explorer/clang-llvmflang-trunk/bin/flang-to-external-fc
```

Comment:
This had not been showing up in the live instance at all since https://github.com/llvm/llvm-project/commit/4841d70a4b7d7cd8c492c16a9da339ec75bca135 removed this script.